### DISCUSSION TITLE
Docs scrollbar stays light in dark mode

### DISCUSSION THREAD

**Discussion by clicktodev** [remix-run/remix#4172] [Proposals]
Could easily be fixed with `color-scheme: dark;` on the html element but I can't find where to put it.
Looked at the docs [source code](https://github.com/remix-run/remix/tree/main/docs) but all I see are md files and no style files.

**Comment by kiliman** [comment]
Only the content is in the repo. The actual docs site is not currently open source. Perhaps someone from the team will see this and update it.
